README.md

```diff
@@ -379,6 +379,7 @@ attribute.
 <textarea data-testid="textarea" required></textarea>
 <div data-testid="supported-role" role="tree" required></div>
 <div data-testid="supported-role-aria" role="tree" aria-required="true"></div>
+<div data-testid="without-aria-role" aria-required="true"></div>
```

Comment:
I'm not sure that it should be supported in any element with `aria-required`. It should still validate that it is used on an element with an aria role, and particularly with a role that supports the `aria-required` attribute.

In particular, check the [Associated roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-required#associated_roles) section in the MDN documentation:

> Used in roles:
>
> - [checkbox](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/checkbox_role)
> - [combobox](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role)
> - [gridcell](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/gridcell_role)
> - [listbox](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role)
> - [radiogroup](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radiogroup_role)
> - [spinbutton](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/spinbutton_role)
> - [textbox](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/textbox_role)
> - [tree](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tree_role)

So this new functionality should only acknowledge the attribute if the element on which it is applied has one of these roles.

Reply:
@gnapse I have addressed the review changes.  Can you please review it again.